examples/angular_bazel_architect/BUILD.bazel

```diff
@@ -112,6 +112,26 @@ architect_test(
     ],
 )
 
+architect(
+    name = "lint",
+    args = ["frontend:lint"],
+    data = glob([
+        "src/**/*",
+        "e2e/src/**/*",
+    ]) + [
+        "angular.json",
```

Comment:
i keep wondering if we should extract a constant to make these less repetitive across rules, or if it just doesn't matter because the file is so trivial to generate

Reply:
Yeah definitely, though currently I was focused more on getting things “working” and polish after.

Also probably there are some deps that can be removed such as Angular CLI.